armbian/base/build/customize-armbian-rockpro64.sh

```diff
@@ -73,57 +78,70 @@ fi
 # Disable Armbian script on first boot
 rm -f /root/.not_logged_in_yet
 
-# Set root password (either from configuration or random)
-BASE_ROOTPW=${BASE_ROOTPW:-$(< /dev/urandom tr -dc A-Z-a-z-0-9 | head -c32)}
-echo root:${BASE_ROOTPW} | chpasswd
-export HOME=/root
-
 echoArguments "Starting build process."
+echoArguments "Starting build process." > /opt/shift/config/buildargs.log
 
 set -ex
 
 # Prevent interactive prompts
 export DEBIAN_FRONTEND=noninteractive
 export APT_LISTCHANGES_FRONTEND=none
 export LANG=C LC_ALL="en_US.UTF-8"
+export HOME=/root
 
-# Customize MOTD
-echo "MOTD_DISABLE='header tips updates armbian-config'" >> /etc/default/armbian-motd
-cat << EOF > /etc/update-motd.d/20-shift
-#!/bin/bash
-. /etc/os-release
-. /etc/armbian-release
-KERNELID=$(uname -r)
-TERM=linux toilet -f standard -F metal "BitBox Base"
-printf '\nWelcome to \e[0;91mARMBIAN\x1B[0m %s %s %s %s\n' "$VERSION $IMAGE_TYPE $PRETTY_NAME $KERNELID"
-if ! mountpoint /mnt/ssd -q; then printf '\n\e[0;91mMounting of SSD failed.\x1B[0m \n'; fi
-echo "Configured for Bitcoin TESTNET"; echo
-EOF
-chmod 755 /etc/update-motd.d/20-shift
 
-echo "${BASE_HOSTNAME}" > /etc/hostname
-hostname -F /etc/hostname
+# USERS & LOGIN-----------------------------------------------------------------
+# - group 'bitcoin' covers sensitive information
+# - group 'system' is used for service users without sensitive privileges
+# - user 'root' is locked for login
```

Comment:
(Optional) Perhaps "disabled from logging in with password" would be more clear than "locked for login"?

Alternatively, if the aim is to disable the `root` account, perhaps we should follow the recommendations of `man passwd`:
```
           Note that this does not disable the account. The
           user may still be able to login using another
           authentication token (e.g. an SSH key). To
           disable the account, administrators should use
           usermod --expiredate 1 (this set the account's
           expire date to Jan 2, 1970).
```

Reply:
For the moment I'd leave root login with ssh-keys enabled by default (it still can be disabled with the build option `BASE_SSH_ROOT_LOGIN=false`). The danger of locking oneself out, e.g. because the sudo password is unknown, is quite big.